### PR TITLE
Yonra egg Fluff Implant

### PR DESCRIPTION
**Pull request by Cameron653**
Adds Yonra's custom egg & egg laying implant. 

DNM until their Item Application is up & accepted.

App here: https://forum.vore-station.net/viewtopic.php?f=26&t=1222

### CODE REVIEW

File: code/modules/vore/fluffstuff/custom_items_vr.dm [hunk 1]
```diff
@@ -1762,4 +1836,4 @@
 
 /obj/item/weapon/storage/backpack/fluff/stunstaff/New()
 	..()
-	new /obj/item/weapon/melee/baton/fluff/stunstaff(src)
```
Comment (Arokha) [inline code/modules/vore/fluffstuff/custom_items_vr.dm]: What does this have to do with the egg implant?

Comment (Cameron653) [reply code/modules/vore/fluffstuff/custom_items_vr.dm]: I have no idea. I didn't touch those lines, github did.

**Comment by Cameron653** [comment]
DNM, some major complications at the moment. Will remove this tag after it's dealt with.

File: code/modules/vore/fluffstuff/custom_items_vr.dm (hunk 1 above)
Comment (Arokha) [reply code/modules/vore/fluffstuff/custom_items_vr.dm]: Ah it was just a line ending change.

**Comment by Arokha** [comment]
image
Never seen THAT before. :D
